influunt-app/app/scripts/controllers/controladores/configuracao-grupos.js

```diff
@@ -45,7 +45,8 @@ angular.module('influuntApp')
           idJson: UUID.generate(),
           anel: {idJson: $scope.currentAnel.idJson}, 
           tipo: 'VEICULAR',
-          faseVermelhaApagadaAmareloIntermitente: false
+          faseVermelhaApagadaAmareloIntermitente: false,
+          // tempoVerdeSeguranca: $scope.objeto.
```

Comment:
Se isso for ficar comentado mesmo, é melhor remover o comentário e a vírgula no final da linha de cima.